test/bucket_props_codec_eqc.erl

```diff
@@ -137,7 +137,7 @@ linkfun() ->
     ?LET({M,F}, {atom(), atom()}, {linkfun, {modfun, M, F}}).
 
 datatype() ->
-    {datatype, elements([counter, set, map])}.
+    {datatype, elements([counter, set, map, hll])}.
```

Comment:
Had no idea we had this duplicate list here. I guess we can't depend on riak_kv_types for this. Eugh. OK.